robot/EDA/resources/EDA.py

```diff
@@ -559,3 +559,32 @@ def verify_app_exists(self,app):
         """Verifies that the given app is present in the app launcher"""
         locator=eda_lex_locators["eda_settings"]["app_tile"].format(app)
         self.selenium.wait_until_page_contains_element(locator,timeout=60,error=f'{app} did not open in 1 min')
+
+    @capture_screenshot_on_error 
+    def checkbox_status(self,cbx_name,status):
```

Comment:
Is this keyword being used? I did a quick grep of the code and I'm not seeing where it's being used anywhere.

Reply:
no. I jumped the gun on committing due to the iframe change, messed up on not finishing this. 🤦‍♀